app/src/main/java/org/jellyfin/mobile/model/sql/entity/ServerEntity.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.model.sql.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.HOSTNAME
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.TABLE_NAME
+
+@Entity(tableName = TABLE_NAME, indices = [Index(value = arrayOf(HOSTNAME), unique = true)])
+data class ServerEntity(
+    @PrimaryKey(autoGenerate = true)
+    @ColumnInfo(name = SERVER_ID) val id: Long,
```

Comment:
```suggestion
    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = SERVER_ID) val id: Long,
```

or 

```suggestion
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = SERVER_ID)
    val id: Long,
```

Right now it looks like primarykey doesn't belong to anything